### PR TITLE
[v7r2] Move dirac-install.py

### LINKED ISSUE #4860: Move dirac-install to management repo
From discussions under https://github.com/DIRACGrid/DIRAC/pull/4858, and to decouple from it:
see discussions starting in https://github.com/DIRACGrid/DIRAC/pull/4858#issuecomment-740599481

### PR DESCRIPTION
**Pull request by chrisburr**
Part of #4860.

Leaving as draft for now while waiting for https://github.com/DIRACGrid/management/pull/32

BEGINRELEASENOTES

*Core
CHANGE: dirac-install.py should now be taken from management 

ENDRELEASENOTES

### CODE REVIEW

File: Core/scripts/dirac-install.py [hunk 1]
```diff
@@ -1,2724 +1,49 @@
...
+sys.stderr.write("\n")
+sys.stderr.write("Please update your scripts to use:\n")
+sys.stderr.write("    https://raw.githubusercontent.com/DIRACGrid/management/add-dirac-install/dirac-install.py\n")
+sys.stderr.write("\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+sys.stderr.write("#" * 100 + "\n")
+
```
Comment (andresailer) [inline Core/scripts/dirac-install.py:29]: Fail if the environment variable DIRAC_DEPRECATED_FAIL is set

Comment (chrisburr) [reply Core/scripts/dirac-install.py:29]: Done

**Comment by chrisburr** [comment]
> Integration tests / Integration (Python 3 client, Yes) (pull_request)

This will fail until #4989 is merged.

**Comment by fstagni** [comment]
This does not fully fixes https://github.com/DIRACGrid/DIRAC/issues/4860 right?
In the end we want to fully remove these files, and we should remove its references to it. This is of course a step in the right direction but you should not have the "fixes #4860" on top of it.

**Comment by fstagni** [comment]
The integration / python3 test fails within this one.

**Comment by chrisburr** [comment]
> The integration / python3 test fails within this one.

https://github.com/DIRACGrid/DIRAC/pull/4994#issuecomment-785797441

**Review by fstagni — APPROVED** [review]

**Review by fstagni — APPROVED** [review]

**Review by atsareg — APPROVED** [review]

**Pull request merged**